doc/policies/admin.rst

```diff
@@ -110,7 +110,9 @@ Tokens can have additional token information, which can be
 viewed in the :ref:`token_details`.
 
 If the ``set`` action is defined, the administrator allowed
-to set those token information.
+to set those token properties like description, max_failcount
```

Comment:
... the administrator **is** allowed ...
Also please mark the properties as code

Reply:
done in 9094bc5e

Reply:
Fixed this in commit.